### PR TITLE
Compute vm snapshot policy attachment (issue 1462)

### PR DESCRIPTION
**Pull request by eandresr**
Here we able to configure disk snapshots, for just boot (or not), or other attached disks, each one can have it's own.

### CODE REVIEW

File: modules/compute-vm/variables.tf [hunk 1]
```diff
@@ -35,14 +35,21 @@ variable "attached_disk_defaults" {
   }
 }
 
+variable "boot_disk_snapshot_policy" {
```
Comment (ludoo) [inline modules/compute-vm/variables.tf:38]: can we move this to the `boot_disk` variable?

Comment (eandresr) [reply modules/compute-vm/variables.tf:38]: much better!

File: modules/compute-vm/main.tf [hunk 2]
```diff
@@ -116,6 +116,44 @@ resource "google_compute_region_disk" "disks" {
   }
 }
 
+# Disks snapshot handler
+data "google_compute_instance" "instance" {
```
Comment (ludoo) [inline modules/compute-vm/main.tf:120]: Why do you need a data source?

**Comment by ludoo** [comment]
I am closing this, implementing this in the module should support defining the policy with support for instance and snapshot schedule, and avoid using datasources. I will try and send a PR for this in the next few days.

**Pull request closed without merging**